www/css/ionic.app.css

```diff
@@ -9097,6 +9097,37 @@ a.button {
   max-width: 100%;
   max-height: 10em; }
 
+.tabulator .button {
+  min-height: 0;
+  padding: 0;
+  margin: 0; }
+
+.tabulator .calc-row {
+  height: 20%;
+  min-height: 0;
+  padding: 0;
+  margin: 0; }
+  .tabulator .calc-row .button {
+    height: 100%; }
+
+.tabulator .section-1 {
+  height: 40%; }
+
+.tabulator .section-2 {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 60%; }
+
+.calc-component {
+  display: block;
+  height: 50%; }
+
+.number-display {
+  text-align: right;
+  font-size: 2em; }
+
```

Comment:
This file and the minified version are actually in the .gitignore file so they should be removed from the repo. Try `git rm --cached www/css/ionic.app.*`. Then they won't keep showing up all large and in charge in the PRs.